README.md

```diff
@@ -206,6 +206,32 @@ When using a GitHub personal access token, the username is arbitrary. GitHub doe
 
 If there are no credentials present in the URL and authentication is needed, the [Git credential helper](https://git-scm.com/doc/credential-helpers) will be queried.
 
+### HTTP authentication
+
+uv supports credentials over HTTP when querying package registries.
+
+Authentication can come from the following sources, in order of precedence:
+
+- The URL e.g. `https://<user>:<password>@<hostname>/...`
```

Comment:
Sorry to do this but elsewhere I always write this like `The URL, e.g., `. Can we just keep that for consistency even if you don't agree?

Reply:
I hate it but will oblige you